CMakeLists.txt

```diff
@@ -204,12 +211,9 @@ if(SOFA_INSTALL_RESOURCES_FILES)
     ## Install resource files
     install(DIRECTORY share/ DESTINATION share/sofa)
     install(DIRECTORY examples/ DESTINATION share/sofa/examples)
-    # Create etc/sofa.ini: it contains the paths to share/ and examples/. In the
-    # build directory, it points to the source tree, whereas in the install
-    # directory, it contains to relative paths to the installed resource directory.
-    set(SHARE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/share")
-    set(EXAMPLES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/examples")
-    configure_file(${SOFA_KERNEL_SOURCE_DIR}/etc/sofa.ini.in "${CMAKE_BINARY_DIR}/etc/sofa.ini")
+	
+	# Create etc/installedSofa.ini: it contains the paths to share/ and examples/. Compare to the Sofa.ini in the
+	# build directory, this one contains relative paths to the installed resource directory.
     set(SHARE_DIR "../share/sofa")
     set(EXAMPLES_DIR "../share/sofa/examples")
     configure_file(${SOFA_KERNEL_SOURCE_DIR}/etc/sofa.ini.in "${CMAKE_BINARY_DIR}/etc/installedSofa.ini")
```

Comment:
Why didn't you also remove this part from the `if(SOFA_INSTALL_RESOURCES_FILES)` block?
If we install with `SOFA_INSTALL_RESOURCES_FILES = false`, _src/etc/installedSofa.ini_ won't be created and _build/install/etc/sofa.ini_ neither.